tests/test_cat.py

```diff
@@ -23,14 +23,14 @@ def test_init_catalog(thredds_cat_url):
 def test_entry(thredds_cat_url):
     cat = intake.open_thredds_cat(thredds_cat_url)
     entry = cat['err.mnmean.v3.nc']
-    assert isinstance(entry, intake.catalog.local.LocalCatalogEntry)
+    assert isinstance(entry, intake.source.base.DataSource)
     d = entry.describe()
     assert d['name'] == 'err.mnmean.v3.nc'
     assert d['container'] == 'xarray'
     assert d['plugin'] == ['netcdf']
     assert (
         d['args']['urlpath']
-        == 'https://psl.noaa.gov/psd/thredds/dodsC/Datasets/noaa.ersst/err.mnmean.v3.nc'
+        == 'https://psl.noaa.gov/thredds/dodsC/Datasets/noaa.ersst/err.mnmean.v3.nc'
```

Comment:
For the future, we may wish to run a local thredds service instead of a real connection, e.g., https://github.com/Unidata/thredds-docker